products/extensions/b2b-components/index.md

```diff
@@ -20,3 +20,134 @@ In the world of digital B2B commerce, where businesses engage with other compani
 * **Quick Order and Shopping List** takes care of distinctive B2B buying behaviors.
 
 * **Digital Sales Composables** aims to provide a set of composable frontends to cover more complex Sales Representative jobs.
+
+## Configuring custom toggles for B2B components
+
+The B2B Components allow merchants to selectively choose and configure B2B capabilities according to their needs. They offer merchants the ability to craft a tailored B2B ecommerce experience for their business partners while also allowing agencies to fine-tune Shopware to meet specific requirements. This means that B2B Components can be individually activated or deactivated for each business partner within the shop.
+
+The following articles will guide you how to do this by creating custom toggles via a plugin for B2B Components (Customer-specific features).
+
+The **Customer-specific features** section on the Customer detail page allows the shop merchant to turn these B2B features on or off for a specific customer. 
+
+![Feature Toggles](../../../assets/b2b-feature-toggles.png)
+
+To achieve this, ACL and address the following cases where functionality may be hidden:
```

Comment:
May I know what this sentence mean ?

Reply:
I believe we need to clarify two points: 

The first one is Customer-specific features: Enable Merchants to show **specific customers** using B2B components. 

The second point is the ACL, which enables merchants to define permissions for **shop users** to Read, Update, Delete, Create, with specific B2B components.